downstream/modules/automation-hub/proc-create-content-developers.adoc

```diff
@@ -4,33 +4,33 @@
 
 = Creating a new group for content curators
 
-You can create a new group in Automation Hub designed to support content curation in your organization who will contribute internally developed collections for publication in Automation Hub.
+You can create a new group in {HubName} designed to support content curation in your organization that contributes internally developed collections for publication in {HubName}.
 
-In this section you will create a new group and assign the required permissions to help content developers create namespaces and upload their collections to Automation Hub.
+This section shows you how to create a new group and assign the required permissions to help content developers create namespaces and upload their collections to {HubName}.
 
 .Prerequisites
 
-* You have *admin* permissions in Automation Hub and create groups.
+* You have *admin* permissions in {HubName} and create groups.
```

Comment:
Is it administrative, or administration? Or is admin the name given on the interface?

Reply:
> Is it administrative, or administration? Or is admin the name given on the interface?

Is there a way that we can confirm this UI?